Docs/source/install/hpc/hpc3.rst

```diff
@@ -0,0 +1,171 @@
+For example, if you are member of the project ``strawberry``, then run ``vi $HOME/hpc3_gpu_warpx.profile`` and edit line 2 to read:
+
+.. code-block:: bash
+
+   export proj="strawberry"
+
+Exit the ``vi`` editor with ``Esc`` and then type ``:wq`` (write & quit).
+
+.. important::
+
+   Now, and as the first step on future logins to HPC3, activate these environment settings:
+
+   .. code-block:: bash
+
+      source $HOME/hpc3_gpu_warpx.profile
+
+Finally, since HPC3 does not yet provide software modules for some of our dependencies, install them once:
+
+.. code-block:: bash
+
+   bash $HOME/src/warpx/Tools/machines/hpc3-uci/install_gpu_dependencies.sh
+
+.. dropdown:: Script Details
+   :color: light
+   :icon: info
+   :animate: fade-in-slide-down
+
+   .. literalinclude:: ../../../../Tools/machines/hpc3-uci/install_gpu_dependencies.sh
+      :language: bash
+
+
+.. _building-hpc3-compilation:
+
+Compilation
+-----------
+
+Use the following :ref:`cmake commands <building-cmake>` to compile:
+
+.. code-block:: bash
+
+   cd $HOME/src/warpx
+   rm -rf build
+
+   cmake -S . -B build -DWarpX_COMPUTE=CUDA -DWarpX_PSATD=ON -DWarpX_LIB=ON -DWarpX_DIMS="1;2;RZ;3"
+   cmake --build build -j 16
+   cmake --build build -j 16 --target pip_install
+
+**That's it!**
+The WarpX application executables are now in ``$HOME/src/warpx/build/bin/`` and we installed the ``pywarpx`` Python module.
+
+Now, you can :ref:`submit HPC3 compute jobs <running-cpp-hpc3>` for WarpX :ref:`Python (PICMI) scripts <usage-picmi>` (:ref:`example scripts <usage-examples>`).
+Or, you can use the WarpX executables to submit HPC3 jobs (:ref:`example inputs <usage-examples>`).
+For executables, you can reference their location in your :ref:`job script <running-cpp-hpc3>` or copy them to a location in ``$PSCRATCH``.
+
+
+.. _building-hpc3-update:
+
+Update WarpX & Dependencies
+---------------------------
+
+If you already installed WarpX in the past and want to update it, start by getting the latest source code:
+
+.. code-block:: bash
+
+   cd $HOME/src/warpx
+
+   # read the output of this command - does it look ok?
+   git status
+
+   # get the latest WarpX source code
+   git fetch
+   git pull
+
+   # read the output of these commands - do they look ok?
+   git status
+   git log # press q to exit
+
+And, if needed,
+
+- :ref:`update the hpc3_gpu_warpx.profile file <building-hpc3-preparation>`,
+- log out and into the system, activate the now updated environment profile as usual,
+- :ref:`execute the dependency install scripts <building-hpc3-preparation>`.
+
+As a last step, clean the build directory ``rm -rf $HOME/src/warpx/build`` and rebuild WarpX.
+
+
+.. _running-cpp-hpc3:
+
+Running
+-------
+
+The batch script below can be used to run a WarpX simulation on multiple nodes (change ``-N`` accordingly) on the supercomputer HPC3 at UCI.
+This partition as up to `ABC nodes <https://rcic.uci.edu/hpc3/slurm.html#memmap>`__ with four V100 GPUs per node.
```

Comment:
@floresv299 let's complete this based on https://github.com/ECP-WarpX/WarpX/issues/3938#issuecomment-1597467944
```suggestion
This partition as up to `N nodes <https://rcic.uci.edu/hpc3/slurm.html#memmap>`__ with four V100 GPUs per node.
```